applications/fishing-map/src/features/sidebar/heatmaps/HeatmapsSection.tsx

```diff
@@ -1,17 +1,29 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import { useSelector } from 'react-redux'
 import { IconButton } from '@globalfishingwatch/ui-components'
 import { selectTemporalgridDataviews } from 'features/workspace/workspace.selectors'
 import styles from 'features/sidebar/common/Sections.module.css'
+import { useDataviewInstancesConnect } from 'features/workspace/workspace.hook'
 import LayerPanel from './HeatmapLayerPanel'
 
 function HeatmapsSection(): React.ReactElement {
   const dataviews = useSelector(selectTemporalgridDataviews)
+  const { removeDataviewInstance } = useDataviewInstancesConnect()
+  const onAddClick = useCallback(() => {
+    removeDataviewInstance('fishing-1')
```

Comment:
Unclear why `onAddSomething` triggers `removeSomething`.
Maybe `toggleDataviewInstance` ?

Reply:
For now, this is just a workaround that will be deleted later but something we need to discuss is how to remove an existing dataviewInstance from the workspace...

Reply:
Please add a comment/TODO.